src/rules/no-focused-tests.ts

```diff
@@ -80,16 +95,53 @@ export default createRule({
           callee.object.type === AST_NODE_TYPES.MemberExpression &&
           isCallToTestOnlyFunction(callee.object)
         ) {
+          const calleeObject: TSESTree.MemberExpression = callee.object;
```

Comment:
```suggestion
          const calleeObject = callee.object;
```

Is the type annotation needed?

Reply:
can probably move the assignment above the `if` and it should be narrowed properly?

Reply:
Well if I do not use the `calleeObject` variable I have some type errors on properties used below, but it works without the type.